### PR TITLE
WIP - To be merged for 7.1 MTA-4016: Analyzer task RBAC

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTA-4016](https://issues.redhat.com/browse/MTA-4016)

### PREVIEW

* [Red Hat Single Sign-On](https://anarnold97.github.io/MTA-Preview/MTA/MTA-4016-Analyzer-task-RBAC.html#mta-7-red-hat-single-sign-on_user-interface-guide)

* [Controlling MTA tasks by using Task Manager](https://anarnold97.github.io/MTA-Preview/MTA/MTA-4016-Analyzer-task-RBAC.html#mta-web-controlling-tasks-with-task-manager_user-interface-guide)

### CODE REVIEW

**Comment by anarnold97** [comment]
@mguetta - please can you have a look, as i am not sure what else to add to the snippet, or where else this should be added?
Thanks

File: docs/topics/analyzer-rbac-snippet.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// snippet
+
+[WARNING]
+.No multi-user access restrictions on resources
+====
+There are no multi-user access restrictions on resources.
+
+An analyzer task created by a user, can be canceled by any other user.
```
Comment (istein1) [inline docs/topics/analyzer-rbac-snippet.adoc:8]: I suggest to remove the specific example of analyzer task access,
as we have the no multi-user access restriction on other resources as well.
Just leave the ".No multi-user access restrictions on resources" warning
and place it in the head of the MTA doc.

**Review by mguetta1 — CHANGES_REQUESTED** [review]
@anarnold97 thanks for this PR.
Could you please add this as a known issue rather than snippet?
And I am not sure about including this information in all the files here. From my point of view, I would like to see it as a known issue with the example that "An analyzer task created by a user, can be canceled by any other use" as a result

**Comment by anarnold97** [comment]
> @anarnold97 thanks for this PR. Could you please add this as a known issue rather than snippet? And I am not sure about including this information in all the files here. From my point of view, I would like to see it as a known issue with the example that "An analyzer task created by a user, can be canceled by any other use" as a result

@mguetta1 - i will add as a **Known Issue** in another PR, so we are not relying on the readers looking at the release notes. See - [MTA-4014: Custom rules in analysis Dependency rule is not fired](https://github.com/migtools/mta-documentation/pull/74)  

* [7.1.1 Known issues](https://anarnold97.github.io/MTA-Preview/MTA/MTA-4014.html#known-issues-7-1-1_release-notes)

* [7.1.0 Known issues](https://anarnold97.github.io/MTA-Preview/MTA/MTA-4014.html#known-issues-7-1-0_release-notes)

**Review by dymurray — APPROVED** [review]

File: docs/cli-guide/master.adoc [hunk 2]
```diff
@@ -19,6 +19,8 @@ include::topics/making-open-source-more-inclusive.adoc[]
 // About the {UserCLIBookName}
 include::topics/about-cli-guide.adoc[leveloffset=+2]
 
+include::topics/analyzer-rbac-snippet.adoc[]
+
```
Comment (anarnold97) [inline docs/cli-guide/master.adoc:23]:
```suggestion
```

File: docs/eclipse-code-ready-studio-guide/master.adoc [hunk 3]
```diff
@@ -20,6 +20,9 @@ include::topics/making-open-source-more-inclusive.adoc[]
 == Introduction
 
 include::topics/about-ide-addons.adoc[leveloffset=+2]
+
+include::topics/analyzer-rbac-snippet.adoc[]
+
```
Comment (anarnold97) [inline docs/eclipse-code-ready-studio-guide/master.adoc:25]:
```suggestion
```

File: docs/getting-started-guide/master.adoc [hunk 4]
```diff
@@ -19,6 +19,8 @@ include::topics/making-open-source-more-inclusive.adoc[]
 // About the {IntroToMTABookName}
 include::topics/mta-about-the-intro-to-mta-guide.adoc[leveloffset=+1]
 
+include::topics/analyzer-rbac-snippet.adoc[]
+
```
Comment (anarnold97) [inline docs/getting-started-guide/master.adoc:23]:
```suggestion
```

File: docs/plugin-guide/master.adoc [hunk 5]
```diff
@@ -18,6 +18,8 @@ include::topics/making-open-source-more-inclusive.adoc[]
 // About the IDE Plugin Guide
 include::topics/plugin-intro.adoc[leveloffset=+2]
 
+include::topics/analyzer-rbac-snippet.adoc[]
+
```
Comment (anarnold97) [inline docs/plugin-guide/master.adoc:22]:
```suggestion
```

File: docs/intellij-idea-plugin-guide/master.adoc [hunk 6]
```diff
@@ -22,6 +22,8 @@ include::topics/making-open-source-more-inclusive.adoc[]
 // About the IntelliJ IDEA plugin
 include::topics/about-ide-addons.adoc[leveloffset=+2]
 
+include::topics/analyzer-rbac-snippet.adoc[]
+
```
Comment (anarnold97) [inline docs/intellij-idea-plugin-guide/master.adoc:26]:
```suggestion
```

File: docs/rules-development-guide/master.adoc [hunk 7]
```diff
@@ -20,6 +20,8 @@ include::topics/making-open-source-more-inclusive.adoc[]
 // About the Rules Development Guide
 include::topics/rules-guide-intro.adoc[leveloffset=+2]
 
+include::topics/analyzer-rbac-snippet.adoc[]
+
```
Comment (anarnold97) [inline docs/rules-development-guide/master.adoc:24]:
```suggestion
```

File: docs/vs-code-extension-guide/master.adoc [hunk 8]
```diff
@@ -20,6 +20,8 @@ include::topics/making-open-source-more-inclusive.adoc[]
 // About the VS Code extension
 include::topics/about-ide-addons.adoc[leveloffset=+2]
 
+include::topics/analyzer-rbac-snippet.adoc[]
+
```
Comment (anarnold97) [inline docs/vs-code-extension-guide/master.adoc:24]:
```suggestion
```

File: docs/topics/analyzer-rbac-snippet.adoc [hunk 9]
```diff
@@ -0,0 +1,7 @@
+// snippet
+
+[WARNING]
+.No multi-user access restrictions on resources
+====
+There are no multi-user access restrictions on resources.
```
Comment (anarnold97) [inline docs/topics/analyzer-rbac-snippet.adoc:6]:
```suggestion
There are no multi-user access restrictions on resources. For example, an analyzer task created by a user, can be canceled by any other user.
```

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 10]
```diff
@@ -8,6 +8,8 @@
 
 You can analyze more than one application at a time against more than one transformation target in the same analysis.
 
+include::analyzer-rbac-snippet.adoc[]
+
```
Comment (anarnold97) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:12]:
```suggestion
```

**Comment by istein1** [comment]
LGTM

File: docs/topics/analyzer-rbac-snippet.adoc [hunk 11]
```diff
@@ -0,0 +1,7 @@
+// snippet
+
+[WARNING]
+.No multi-user access restrictions on resources
+====
+There are no multi-user access restrictions on resources. For example, an analyzer task created by a user, can be canceled by any other user.
```
Comment (anarnold97) [inline docs/topics/analyzer-rbac-snippet.adoc:6]:
```suggestion
There are no multi-user access restrictions on resources. For example, an analyzer task created by a user can be canceled by any other user.
```

**Pull request merged**